tests/credentials_template.json

```diff
@@ -0,0 +1,12 @@
+// This file can be used in local development by specifying needed credentials and other options for the tests.
+// Copy it with new name "credentials.json" and tests will look in it."
+
+// NOTE: also remove those invalid comments.
+{
+  "scalyr_api_key": "",
```

Comment:
I assume you used JSON and not YAML since I just removed pyyaml dependency the other day since it was unused? :)

Reply:
I guess I just didn't see any problem with using json this time :) I even didn't touch existing similar YAML config in our other tests, since I really want to move everything in new end to end tests and them gone.